README.md

```diff
@@ -8,10 +8,19 @@ Repository to audit, service, and clean up leftover AWS resources
 
 ## Deploying Shredder Locally
 
-Load up CRC or Minishift. If you don't already have them, create the namespace for `aws-account-operator` and `aws-account-shredder`.
```

Comment:
Is there a reason to change this? I personally like the way it was before plus it has the benefit of easy copy+paste

Reply:
Ha I made the changes to make it easier to copy+paste the commands :)

Reply:
You are totally right, I meant I like the way you currently have it over the way it was before. My bad I just got really confused...